frontend/src/features/Mission/components/MissionForm/ActionsTimeLine/index.tsx

```diff
@@ -120,14 +120,22 @@ export function ActionsTimeLine({ currentActionId, setCurrentActionId }) {
   )
 
   const handleDuplicateAction = useCallback(
-    id => {
+    (e, id) => {
+      e.stopPropagation()
       const envAction = envActions && envActions.find(action => action.id === id)
 
-      if (envAction) {
-        const duplicatedAction = actionFactory(envAction)
-        setFieldValue('envActions', [duplicatedAction, ...(envActions || [])])
-        setCurrentActionId(undefined)
+      if (!envAction) {
+        return
       }
+
+      let newAction = { ...envAction }
+      if ('reportingIds' in newAction) {
```

Comment:
suggestion : je pense qu'on peut mettre la condition avant le spread operator. ça montre que tu regardes que l'envaction de base a des reportings et pas la nouvelle (meme si c'est la meme chose).

```suggestion
      if ('reportingIds' in envAction) {
      let newAction = {...envaction, reportingIds: []}
```